### DISCUSSION TITLE
Edit Photo : Entity Not Found

### DISCUSSION THREAD

**Discussion by danmed** [photoprism/photoprism#2161] [Q&A] [answered]
When i select a photo and go to the bubble menu in the bottom right and select "Edit" the edit page opens but with no details and a red pop up at the bottom saying "Entity not found".

If i expand the photo and then click "Edit" at the top, it works fine...

I'm new to Photoprism so if this is a known bug i apologise..

Am i doing something wrong here?

![image](https://user-images.githubusercontent.com/3878490/158803566-26b59ad9-d880-4d62-8f4a-499a2769534f.png)

**Comment by lastzero** [comment, marked as answer]
Other items may also be selected in the clipboard. Are you sure that it is only the current image? Any updates running in the background, like indexing/importing?

**Reply by danmed** [reply]
See this is strange.. i've got nothing selected.. yet the bubble has the number 20 on it....

**Reply by lastzero** [reply]
Then there are 20 items selected, some of which may be deleted or not visible in your current view. Clear the clipboard using the X button at the top of the expanded menu 👍  Should be described somewhere in our docs as well...

**Reply by danmed** [reply]
Thank you.. that was the issue.
